### ISSUE TITLE
Typegen: The method names in OperationMethods are not escaped properly

### ISSUE THREAD

**Issue by alexkar598** [anttiviljami/openapi-client-axios#26]
Typegen output code breaks if an operationid contains a special character like .

![sad code](https://i.imgur.com/L7TpPf4.png)

Can be fixed by by making operationMethod be
```javascript
var operationMethod =
        methodName.replace(/[\.,-\/#!$%\^&\*;:{}=\-_`~()\+\s]/g,'_') +
        '(\n' +
        operationArgs
            .map(function (arg) {
                return indent_string_1.default(arg, 2);
            })
            .join(',\n') +
        '  \n): ' +
        returnType;```

inside generateMethodForOperation

I would make a PR for it but it seems the repo went into oblivion

**Comment by anttiviljami** [comment]
We should come up with a common way to sanitize operationIds. The code you suggested (replace all special characters with _) would be a major breaking change.

**Comment by anttiviljami** [comment]
Fixed in `3.7.0`
